projector-plugin/src/main/kotlin/org/jetbrains/projector/plugin/ProjectorService.kt

```diff
@@ -272,5 +295,6 @@ class ProjectorService : PersistentStateComponent<ProjectorConfig> {
 
   override fun loadState(state: ProjectorConfig) {
     config = state
+    setSystemProperty(ProjectorServer.ENABLE_CONNECTION_CONFIRMATION, if (config.confirmConnection) "true" else "false")
```

Comment:
Was before this PR but can be improved now (here and in other places):

```suggestion
    setSystemProperty(ProjectorServer.ENABLE_CONNECTION_CONFIRMATION, config.confirmConnection.toString())
```